harness/determined/tensorboard/base.py

```diff
@@ -22,11 +23,17 @@ def __init__(
     ) -> None:
         self.base_path = base_path
         self.sync_path = sync_path
-        self._synced_event_sizes: Dict[pathlib.Path, int] = {}
+        self.last_sync = float(0)
```

Comment:
I think this is a bit more idiomatic:
```suggestion
        self.last_sync = 0.0
```

Reply:
:facepalm: